bc_obps/service/tests/test_operation_service.py

```diff
@@ -106,62 +106,6 @@ def test_assigns_single_selected_purpose():
         assert operation.updated_at is not None
         assert operation.registration_purpose == Operation.Purposes.REPORTING_OPERATION
 
-    @staticmethod
```

Comment:
I think we should keep/repurpose one of these to confirm that the detail record is deleted

Reply:
This test, `test_old_purpose_opted_in`, covers our it. However, I can create an additional test if you have any concerns.

Reply:
Right, all good then!